spec/src/main/asciidoc/architecture.asciidoc

```diff
@@ -844,8 +863,8 @@ PublisherBuilder<O> method(I payload)
 @Outgoing("out") 
 Message<O> method(Message<I> msg)
 ----
-| Post-Processing
-| None, Manual, Pre-Processing, Post-Processing
+| Pre-Processing
```

Comment:
Cross check against "When dealing with single payloads, the POST_PROCESSING strategy is the default strategy."

Reply:
Fine noyt a single payload